Sources/LeafKit/LeafRenderer.swift

```diff
@@ -66,23 +109,20 @@ public final class LeafRenderer {
         return try serializer.serialize()
     }
 
-    private func expand(template: String) -> String {
-        var path = template
```

Comment:
Obviated for NIOFiles - LeafRenderer shouldn't be attempting to set a full path, as other sources for templates (eg database backs) would not necessarily recognize path structure in this way